src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OperatorsShouldHaveSymmetricalOverloads.cs

```diff
@@ -16,7 +18,10 @@ public abstract class OperatorsShouldHaveSymmetricalOverloadsAnalyzer : Diagnost
 
```

Comment:
Since the analyzer is not doing any language specific operation in `BasicOperatorsShouldHaveSymmetricalOverloadsAnalyzer` and `CSharpOperatorsShouldHaveSymmetricalOverloadsAnalyzer` can be deleted and `OperatorsShouldHaveSymmetricalOverloadsAnalyzer ` can be made `sealed`